### ISSUE TITLE
Warhead activation sends wrong message in discord (0 seconds but it was 89)

### ISSUE THREAD

**Issue by LaFesta1749** [KarlOfDuty/SCPDiscord#152]
When someone activates the nuke it sends the wrong message to discord.

This is the discord message:
```The alpha warhead has been activated by petar_mndz (76561199265659720), 0 seconds until detonation. All personnel must evacuate immediately.```

This is the actual timer (or it was when I took the picture):
![image](https://github.com/KarlOfDuty/SCPDiscord/assets/1494298/bdcce01e-4919-4537-83ac-3a296efd9242)

**Comment by KarlOfDuty** [comment]
Yes, this is a bug in the game, not in the plugin. I have reported it to them but they have not fixed it yet.

**Comment by KarlOfDuty** [comment]
https://github.com/northwood-studios/NwPluginAPI/issues/144

**Comment by Griseo-bh3rd** [comment]
Maybe add a short delay to fix the problem?

**Comment by KarlOfDuty** [comment]
No, the parameter never works.

**Comment by KarlOfDuty** [comment]
Seems to be fixed in the game now.
